tests/unit/test_manifest_Manifest.py

```diff
@@ -1,53 +1,96 @@
 import json
+import sys
 from pathlib import Path
 from unittest import TestCase
 from unittest.mock import patch
 
-from ledgerwallet.manifest import AppManifest
+if sys.version_info >= (3, 11):
+    import tomllib
+else:
+    import toml as tomllib
+
+from ledgerwallet.manifest_json import AppManifestJson
+from ledgerwallet.manifest_toml import AppManifestToml
 
 
 class ManifestTest(TestCase):
```

Comment:
For clarity, I would prefer testing `AppManifestJson` and `AppManifestToml` in their own test set, but I don't have a strong opinion on that.